### PR TITLE
Add github workflow to lint files

### PR DESCRIPTION
**Pull request by chrichri17**
This PR adds a jobs that run on every PR to lint the frontend files.

What's next ?
------------
- [x] add `black` linter for the backend

### CODE REVIEW

**Comment by chrichri17** [comment]
Current dependencies on/for this PR:
* main
  * **PR #5** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/5" target="_blank">Graphite</a> 
    * **PR #6** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/6" target="_blank">Graphite</a>  👈
      * **PR #7** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/7" target="_blank">Graphite</a> 
        * **PR #8** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/8" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/YacineSteeve/quizoot/6?utm_source=stack-comment).

File: frontend/src/components/Home.vue [hunk 1]
```diff
@@ -11,7 +11,7 @@ import QuizCard from '@/components/QuizCard.vue';
         </div>
         <!-- Quiz details will be inserted as attributes to QuizCard components
             for each quiz in quizzes list -->
-        <QuizCard v-for="num in [1, 2, 3, 4, 5, 6, 7]" v-bind:index="num" />
+        <QuizCard v-for="num in 7" v-bind:key="num" />
```
Comment (chrichri17) [inline frontend/src/components/Home.vue:14]: drive-by change.

- I'm using `key` instead of `index` because `eslint` was throwing an error. See https://eslint.vuejs.org/rules/require-v-for-key.html
- I simplified the `v-for` expression

Comment (YacineSteeve) [reply frontend/src/components/Home.vue:14]: * I'm okay, `key` is more suitable (thanks to `eslint` !).
* An amazing simplification, by the way ! Didn't know that was possible.

**Review by YacineSteeve — APPROVED** [review]
